### DISCUSSION TITLE
G16 2024 Keyboard Lighting Support

### DISCUSSION THREAD

**Discussion by highlanderc** [seerge/g-helper#2671] [General]
Hello Seerge,

I have contributed, plan on doing more. This program is a game changer for Asus to be honest.

Can you add support for keyboard lighting on 2024 G16. It is currently not responding. This is insignificant, so if its down on your list, I won't hold it against you. 

If there is any data I can gather for you let me know.

Thanks

**Comment by seerge** [comment]
@highlanderc hello. Backlight control on G16 2024 is supported

Please make sure that ``Windows Settings -> Personalization -> Dynamic Lightning`` is disabled. Or it will take over.
https://github.com/seerge/g-helper/wiki/FAQ#backlight-doesnt-change-color-or-stays-blue
